### DISCUSSION TITLE
[Bug]: Imported docker image error

### DISCUSSION THREAD

**Discussion by habr05** [meshtastic/firmware#4988] [Troubleshooting]
### Category

Other

### Hardware

Other

### Firmware Version

2.3.15.e48da6f8

### Description

I successfully created a docker image on Orange Pi 5 and imported it into docker on Raspberry Pi 3B+ using the following command:

`sudo docker import --change 'CMD ["sh",  "-cx", "./meshtasticd -d /home/mesh/data"]' meshtastic-device-latest.tar meshtastic/device:latest`

When I try to run it with `docker run -it -p 4403:4403 meshtastic/device` I get this:

`docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "sh": executable file not found in $PATH: unknown.`

What am I missing?

**Comment by fifieldt** [comment]
Thanks for sending in the report, and sorry noone replied.

I believe this is fixed now, in the later images :)
